tests/fixtures/system.js

```diff
@@ -0,0 +1,2 @@
+export const currentStateKey = 'current_state';
```

Comment:
this is constant, rename they as we excepted for constant naming
`CURRENT_STATE_KEY` and etc...

Reply:
done